modules/core/src/core/transform-shader-utils.js

```diff
@@ -71,6 +65,22 @@ export function updateForTextures({vs, sourceTextureMap, targetTextureVarying, t
   };
 }
 
+// builds and returns an object contaning size uniform for each texture
+export function getSizeUniforms({sourceTextureMap, targetTextureVarying, targetTexture}) {
```

Comment:
Some thoughts here:
* In GLSL 3.00 there is a function for getting the size of a texture in the shader, but in WebGL1 one must pass the size in. It would be cool if we would let shaders always use the GLSL function and only add the uniforms when transpiling older shaders.
* I have a similar issue in the postprocessing library, we should align on solutions, let's brainstorm off-line.

Reply:
This should be added to docs?

Reply:
This is an internal method used only by `Transform` class.